CK.ActivityMonitor/Impl/ActivityMonitorBridge.cs

```diff
@@ -114,7 +115,7 @@ void IActivityMonitorBoundClient.SetMonitor( Impl.IActivityMonitorImpl source, b
                 _targetActualFilter = _bridgeTarget.TargetFinalFilter;
                 if( _pullTargetTopicAndAutoTagsFromTarget )
                 {
-                    source.InitializeTopicAndAutoTags( this._targetMonitor.Topic, _targetMonitor.AutoTags );
+                    source.InitializeTopicAndAutoTags( _targetMonitor.Topic, _targetMonitor.AutoTags ); //TODO: @Spi there is a bug there too, no ?
```

Comment:
Here source may be null.
Basically it will throw a nullref if we do SetMonitor with a nullref and with the option "pullTargetTopicAndAutoTagsFromTarget".